python/python/lance/dataset.py

```diff
@@ -438,6 +439,7 @@ def nearest(
             "metric": metric,
             "nprobes": nprobes,
             "refine_factor": refine_factor,
+            "use_index": use_index,
```

Comment:
Mention this is debug purpose?
Also, should we skip the index on a specific column? 

like `skip_indices=[vector, numeric_col_by_btree]`?

Reply:
This is within nearest so it's always for that column

Reply:
added docstring